fastplotlib/widgets/image_widget/_sliders.py

```diff
@@ -0,0 +1,168 @@
+import os
+from time import perf_counter
+
+from imgui_bundle import imgui, icons_fontawesome_6 as fa
+
+from ...ui import EdgeWindow
+
+
+class ImageWidgetSliders(EdgeWindow):
+    def __init__(self, figure, size, location, title, image_widget):
+        super().__init__(figure=figure, size=size, location=location, title=title)
+        self._image_widget = image_widget
+
+        # whether or not a dimension is in play mode
+        self._playing: dict[str, bool] = {"t": False, "z": False}
+
+        # approximate framerate for playing
+        self._fps: dict[str, int] = {"t": 20, "z": 20}
+        # framerate converted to frame time
+        self._frame_time: dict[str, float] = {"t": 1 / 20, "z": 1 / 20}
+
+        # last timepoint that a frame was displayed from a given dimension
+        self._last_frame_time: dict[str, float] = {"t": 0, "z": 0}
+
+        self._loop = False
+
+        if "RTD_BUILD" in os.environ.keys():
+            if os.environ["RTD_BUILD"] == "1":
+                self._playing["t"] = True
+                self._loop = True
```

Comment:
@clewis7 this is how the video auto-plays upon `ImageWidget` create for docs gallery generation